### PR TITLE
Multiple schemas

### PR DESCRIPTION
**Pull request by juliahayward**

### CODE REVIEW

File: _posts/2020-12-21-multipleSchemas.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+we need another migration, #4 - and that migration now needs to check each schema in turn, and if they are not at the
+right state, re-run migrations #1, #2 and #3 somehow. 
+
+The prospect of errors during the catch-up process complicates things further. What happens if something fails
+while schema `baz` is being brought up to the current state? We have no way of reflecting that in the history table as 
+it stands, and so we would need more SQL to figure out what state it was in. 
+
+So, we can't recommend this approach.
+
+## But that's not the end of the story...
```
Comment (MikielAgutu) [inline _posts/2020-12-21-multipleSchemas.md:121]: I wonder if we can preface this blog with a similar call to action. Something to the effect of 'We've had some users ask about multi-schema deployments. Flyway supports some scenarios but not others. This blog post is our current thinking on the matter.  If you have any feedback for us, post in this GitHub issue.'

**Review by MikielAgutu — APPROVED** [review]

File: _posts/2020-12-21-multipleSchemas.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+---
+
+We've had some users ask recently about multi-schema deployments. There are a number of possible scenarios, some
+of which are supported naturally by Flyway and some not. This blog post summarises the current state of play
+on the matter, but we're keen to make sure we cover popular use cases. If you have any feedback for us, 
+there's an ongoing discussion on [Github Issues](https://github.com/flyway/flyway/issues/2995).
+
+## Configuration
+
+There are two main configuration parameters; `flyway.schemas` and `flyway.defaultSchema`. In this post we'll talk
```
Comment (DoodleBobBuffPants) [inline _posts/2020-12-21-multipleSchemas.md:15]: I think this semi-colon should be a colon? Though I don't remember much from my English lessons way back when 😬

File: _posts/2020-12-21-multipleSchemas.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+Now suppose we have a multi-tenant database where we want all schemas to have an identical structure. Now, we no longer
+need separate sets of scripts - and moreover, we no longer *want* separate sets of scripts, as having a single set will
+enforce that uniformity for us. Instead of iterating over configurations, we can iterate instead over the schemas,
+using the `${flyway:defaultSchema}` placeholder in any script that needs a fully qualified name:
+
+```
+# Powershell
+$schemas = @('foo', 'bar')
+foreach ($schema in $schemas) { flyway migrate -configfFiles="c:\src\config\flyway.conf" -defaultSchema="$schema" }
```
Comment (DoodleBobBuffPants) [inline _posts/2020-12-21-multipleSchemas.md:89]: There's an extra `f` in `-configFiles`

File: _posts/2020-12-21-multipleSchemas.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+flyway.defaultSchema=foo
+flyway.locations=c:\src\migrations\foo\
+```
+
+and then migrating each one in turn:
+
+```
+# Powershell
+$schemas = @('foo', 'bar')
+foreach ($schema in $schemas) { flyway migrate -configfFiles="c:\src\config\$schema.conf" }
```
Comment (DoodleBobBuffPants) [inline _posts/2020-12-21-multipleSchemas.md:67]: There's an extra `f` in `-configFiles`

**Pull request merged**